src/build.gradle

```diff
@@ -17,6 +17,7 @@ plugins {
     id 'idea'
     id "io.spring.dependency-management" version "1.0.11.RELEASE" apply false
     id 'org.springframework.boot' version "${springBootVersion}" apply false
+    id "com.dorongold.task-tree" version "2.1.0"
```

Comment:
Is this for visualization or it has some other use?

Reply:
Correct, just for visualization